frontend/src/styles/components/commentupdateform.css

```diff
@@ -0,0 +1,23 @@
+.button-comment {
+  background-color: transparent;
+  border-width: 0px;
+  line-height: normal;
+  display: flex;
+  align-items: center;
+  padding: 0px;
+}
+
+.comment {
+  margin-right: var(--space-xx-small);
+  height: var(--space-medium);
+}
+
+.icon-container {
+  display: flex;
+}
+
+.length {
```

Comment:
i can't tell but is there extra space after the dot? is the styling applied? if not please remove the space

Reply:
I don't think there's a space